superset-frontend/src/SqlLab/components/AceEditorWrapper/AceStyles.less

```diff
@@ -0,0 +1,22 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+.ace_editor.ace_autocomplete {
+  width: 520px;
```

Comment:
Not sure we need a new file here... this could be shoehorned under the `.ace_editor` selector in `main.less`

Optionally, all o' that could be moved into Emotion (though that comes with the side mission of migrating the ligature settings into the SupersetTheme.

Also, is 520px based on... something? Maybe it could be a percentage?

Reply:
Adding it to `main.less` makes sense to me. That would change autocomplete widths for every instance of ace editor that we have.

I chose 520 based on what looked good to me on-screen. Can't be a percentage because the element is absolute positioned.